models/user.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"github.com/beego/beego/v2/client/orm"
+)
+
+type User struct {
+	Default
+
+	Email             string `orm:"unique;size(128)"`
+	EncryptedPassword string
+}
+
+func init() {
+	orm.RegisterModel(new(User))
+
+}
+
+// AddUser insert a new User into database and returns last inserted Id on success.
+func AddUser(m *User) (id int64, err error) {
```

Comment:
Prefer using the standard CRUD naming:

```suggestion
func CreateUser(m *User) (id int64, err error) {
```

Then there could be `UpdateUser` and `DeleteUser`. Th eonly exception would be `GetUser` instead of `ReadUser` 🙈